ui/StatusQ/src/StatusQ/Controls/StatusInput.qml

```diff
@@ -263,10 +263,11 @@ Item {
         }
 
         statusBaseInput.valid = true
+        const rawText = statusBaseInput.edit.getText(0, statusBaseInput.edit.length)
```

Comment:
You know this will break when the text contains non UTF-8 characters, such as emojis?

Reply:
You need to use a UTF-16 emoji there to prove me wrong :) Such as: "👨🏻‍🍼"